L1Trigger/TrackFindingTracklet/interface/CPUTimer.h

```diff
@@ -0,0 +1,44 @@
+#ifndef _FPGATIMER_H_
+#define _FPGATIMER_H_
+#include <math.h>
+#include <sys/time.h>
+
+class CPUTimer {
+public:
+  CPUTimer() {
+    ntimes_ = 0;
+    ttot_ = 0.0;
+    ttotsq_ = 0.0;
+  };
+  virtual ~CPUTimer(){};
```

Comment:
why do you need a `virtual` destructor ?

Reply:
changed